### PR TITLE
updated installation description

### PR DESCRIPTION
**Pull request by sibirrer**
addresses issue #89

### CODE REVIEW

**Comment by alxogm** [comment]
Hi @sibirrer maybe you can add the installing requirements line also in the [Contributing](https://github.com/LSST-strong-lensing/slsim/blob/main/CONTRIBUTING.rst) instructions?

Thank you for addressing the issue !

**Comment by sibirrer** [comment]
@alxogm I make you here review my PR that hopefully fixes your issue :)

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -30,13 +30,14 @@ Finally add the ``slsim-project`` repository as a *remote*. This will allow you
...
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To run your own version of slsim and making sure that your own development changes are reflected (as well as switcing branches),
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To run your own version of slsim and making sure that your own development changes are reflected (as well as switching branches),
 install the slsim package in development mode with
 
 ::
 
   cd slsim
+  pip install -r requirements.txt
```
Comment (alxogm) [inline CONTRIBUTING.rst:40]: Maybe add --user here as well?

**Review by alxogm — APPROVED** [review]
I only left one suggestion that I think it could help, at least for users in the RSP.

**Review by alxogm — APPROVED** [review]

**Pull request merged**